src/index.ts

```diff
@@ -4,7 +4,11 @@ import { CustomClient } from "./client";
 
 // Create a new client instance
 const client = new CustomClient({
-  intents: [GatewayIntentBits.Guilds],
+  intents: [
+    GatewayIntentBits.Guilds,
+    GatewayIntentBits.GuildMessages,
+    GatewayIntentBits.MessageContent,
+  ],
```

Comment:
@MaXeraph Can you explain to me what intents are? I never fully understood them

Reply:
They are a filter, because you can listen to different events happening in the channel, this helps to narrow down and reduce load on the bot.